snippets/nordic-flpr/nordic-flpr.overlay

```diff
@@ -6,7 +6,3 @@
 &cpuflpr_vpr {
 	status = "okay";
 };
-
-&cpuapp_vevif_tx {
```

Comment:
This could be useful for more complex IPC schemes with more channels required. But I guess in such a scenario, it could be added back as a snippet or an overlay.

Reply:
The snipper just covers remote core boot-up, not any extra things. I think it should be a separate snippet.